library/system/src/lib/yast2/fs_snapshot.rb

```diff
@@ -65,6 +65,9 @@ class FsSnapshot
     LIST_SNAPSHOTS_CMD = "LANG=en_US.UTF-8 /usr/bin/snapper --no-dbus --root=%{root} list"
     VALID_LINE_REGEX = /\A\w+\s+\| \d+/
 
+    # Predefined snapshot cleanup strategies (the user can define custom ones, too)
```

Comment:
Ah a bit confusing as I think it mean, that you can pass to parameter also other strategies, which do not work.